### PR TITLE
Feature xml file and url verification

### PR DESCRIPTION
**Pull request by Melissa-Martinez-XL**
## Ticket

* Adding XML file and verification of the blog URL

## Type of change

* [X] Fix
* [ ] Story
* [ ] Chore

## Description of the change

I added the plugins to add the XML file and redirects the old blog URLs to the new ones.

## Screenshot/Execution

Relevant screenshots or steps to execute a task

## Related PRs
[Links to the related PRs](https://github.com/xmartlabs/xl-blog/pull/51)

### CODE REVIEW

**Comment by Melissa-Martinez-XL** [comment]
I don't know if the redirect plugin is working, because it only works when deployed. I will test it when I merge this branch to the base branch (feature-migrating-to-v5), because, I will be able to verify if the plugin works on [netlify deployment](https://deploy-preview-51--xl-blog.netlify.app/) of that branch.

File: gatsby-node.js [hunk 1]
```diff
@@ -50,11 +50,14 @@ exports.onCreateNode = ({ node, actions }) => {
 }
 
 exports.createPages = async ({actions, graphql, reporter}) => {
-    const {createPage} = actions
-    const tagTemplate = path.resolve("src/templates/tags.js")
-    const categoryTemplate = path.resolve("src/templates/categories.js")
-    const authorTemplate = path.resolve("src/templates/authors.js")
-    const postTemplate = path.resolve("src/templates/post.js")
+    const {createPage} = actions;
+    const {createRedirect} = actions;
```
Comment (gpicart) [inline gatsby-node.js:54]:
```suggestion
    const {createPage, createRedirect } = actions;
```

**Review by gpicart — APPROVED** [review]
Just that small change and it can be merged 😄

File: gatsby-node.js (hunk 1 above)
Comment (Melissa-Martinez-XL) [reply gatsby-node.js]: Done

**Pull request merged**